media/js/firefox/new/experiment-firefox-new-batm-anim.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var cop = new Mozilla.TrafficCop({
+        id: 'experiment_firefox_new_batm_anim',
+        variations: {
+            'xv=batmprivate': 33,   // Control: basic page, no machine
```

Comment:
I've updated to use old-fashioned `v=a` and `v=b` params and Traffic Cop seems to be working (also once I finally fixed the file page on the JS, doh).